src/test/java/org/jenkinsci/plugins/workflow/cps/global/CompilationErrorExceptionTest.java

```diff
@@ -0,0 +1,43 @@
+package org.jenkinsci.plugins.workflow.cps.global;
+
+import com.cloudbees.hudson.plugins.folder.Folder;
+import hudson.model.Run;
+import hudson.plugins.git.GitSCM;
+import jenkins.plugins.git.GitSCMSource;
+import jenkins.plugins.git.GitSampleRepoRule;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.libs.GlobalLibraries;
+import org.jenkinsci.plugins.workflow.libs.LibraryConfiguration;
+import org.jenkinsci.plugins.workflow.libs.SCMRetriever;
+import org.jenkinsci.plugins.workflow.libs.SCMSourceRetriever;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.util.Collections;
+
+@Issue("JENKINS-40109")
+public class CompilationErrorExceptionTest extends Assert {
```

Comment:
Why are you extending `Assert` here?

Reply:
Probably just a copy-paste from elsewhere in the plugin (https://github.com/jenkinsci/workflow-cps-global-lib-plugin/blob/2c83e4735d4fa65e60e04db095d2918ba44cfd5b/src/test/java/org/jenkinsci/plugins/workflow/cps/global/UserDefinedGlobalVariableListTest.java#L24), but yeah it would be good to clean it up.